### ISSUE TITLE
sending shortcut to website when in search mode

### ISSUE THREAD

**Issue by berendho** [philc/vimium#871]
Hi,

This might be a duplicate since it seems an obvious bug but I could not find it in the list.

Here is the issue.

I use duckduckgo.com for my internet searches. After you search something you can select the result with the arrow keys and open the link you have found with the 'o' key. 

Since vimium has already mapped the o key to Vomnibar.activate a search bar will appear. This is behaviour as I would expect and nothing wrong with that. The problem occurs when I start entering a search string in the Vomnibar. Let's say I'm looking for google. After typing the o of 'google' in the vomnibar it wil open the link I have selected in duckduckgo.com. 

In other words the shortcuts to the website are not intercepted when searching in the vomnibar.

**Comment by smblott-github** [comment]
Closing.
Fixed with vomnibar in `iframe`.
